### ISSUE TITLE
Depth-dependent antenna model

### ISSUE THREAD

**Issue by christophwelling** [nu-radio/NuRadioReco#229]
Right now, antenna models in the detector description do not take the changing index of refraction with depth into account. It is possible in principle to write different antenna models into the detector description explicitly, but it's better to automate this process.

My suggestion would be to make the ice model available to the detector object (either by passing it to the creator or storing it in the detector description directly) and have the detector calculate the index of refraction at the antenna and choose the correct antenna model.

**Comment by cg-laser** [comment]
I think this is a good suggestion. Technically it is possible to pass the ice model to the detector in NuRadioMC. When the detector is initialized, we already know the ice model. 

Although I don't think we need this urgently at the moment. For simulation studies is is probably more convenient to directly specify the correct antenna model. Then you also don't depend on some magic. Also technically we would need to make sure that all antenna models are available for different index-of-refractions. 

I originally implemented this for ARIANNA where the depth of the antennas change strongly relative to their total depth due to snow accumulation.

**Comment by christophwelling** [comment]
Agreed, this is definitely low priority at the moment and we can't do anything anyway until we have enough antenna models. This issue is mainly to make sure we don't forget about it.
